components/Login.jsx

```diff
@@ -13,9 +15,13 @@ export default function Login() {
 
   const callbackUrl = router.query.callbackUrl || '/cards'
   return (
-    <>
-      <StyledButton>Start Game</StyledButton>
-      <StyledButton onClick={() => signIn('github', { callbackUrl })}>Log in</StyledButton>
-    </>
+    <StyledWrapper>
+      <StyledH1 $variant="extra">HarryPotter</StyledH1>
+      <StyledH1 $variant="big">Flipwise</StyledH1>
```

Comment:
```suggestion
      <StyledH1 $variant="extra" as="h1">HarryPotter</StyledH1>
      <StyledH1 $variant="big" as="h2">Flipwise</StyledH1>
```
It's not good practice to have more than one `<h1>` on a single page. You can still style things like an `<h1>` without actually using multiple `<h1>` tags. In above suggestion, we're using the same styled component but renders "Flipwise" as an `h2` instead of an `h1`.